It is impossible to build for release due to SVEs

### What version did the issue occur in?

04b4b7d51c2c85bec73f9e516dc51818122975a6

### Description

Outstanding vulnerabilities in a dependency prevent builds on release configuration:
```
/home/runner/work/moff-station-14/moff-station-14/Content.Server.Database/Content.Server.Database.csproj : error NU1901: Warning As Error: Package 'System.Security.Cryptography.Xml' 9.0.0 has a known low severity vulnerability, https://github.com/advisories/GHSA-37gx-xxp4-5rgx [/home/runner/work/moff-station-14/moff-station-14/Content.Server/Content.Server.csproj]
/home/runner/work/moff-station-14/moff-station-14/Content.Server.Database/Content.Server.Database.csproj : error NU1901: Warning As Error: Package 'System.Security.Cryptography.Xml' 9.0.0 has a known low severity vulnerability, https://github.com/advisories/GHSA-w3x6-4m5h-cxqf [/home/runner/work/moff-station-14/moff-station-14/Content.Server/Content.Server.csproj]
```

* https://github.com/advisories/GHSA-37gx-xxp4-5rgx
* https://github.com/advisories/GHSA-w3x6-4m5h-cxqf

I'm only making this issue so that I can get notifications when somebody closes/fixes this so that I can uncomment the "warnings as errors" config in my downstream fork :^)

### Reproduction

`dotnet build --configuration release`

### Additional Context

<img width="1472" height="54" alt="Image" src="https://github.com/user-attachments/assets/6464af12-806e-49db-b112-18b9513d7377" />

https://discord.com/channels/310555209753690112/770682801607278632/1494037439029837835

I imagine this'd effect everything from 04b4b7d51c2c85bec73f9e516dc51818122975a6 up until a fix to bump the dependency is in (or something which turns off warnings as errors).

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -8,11 +8,11 @@
     <PackageVersion Remove="Microsoft.EntityFrameworkCore.Design" />
     <PackageVersion Include="CsvHelper" Version="33.1.0" />
     <PackageVersion Include="ImGui.NET" Version="1.87.3" />
-    <PackageVersion Include="Microsoft.EntityFrameworkCore.Design" Version="10.0.0">
+    <PackageVersion Include="Microsoft.EntityFrameworkCore.Design" Version="10.0.6">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageVersion>
-    <PackageVersion Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="10.0.0" />
+    <PackageVersion Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="10.0.6" />
     <PackageVersion Include="NetCord" Version="1.0.0-alpha.388" />
     <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="10.0.0" />
     <PackageVersion Include="OpenTK" Version="4.9.4" />

